package.json

```diff
@@ -11,11 +11,13 @@
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "three": "^0.162.0"
```

Comment:
Do not use `^` in version.

Reply:
I removed ^ and added file .nprmc with save-exact set to true